lib/doorkeeper/rake/db.rake

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+namespace :doorkeeper do
```

Comment:
I think that rake namespaces should not be affected by this, while I agree that `task` should not be too long.

We are using a rubocop config like this at our projects:

```yml
# We need to configure exemptions for blocks that we generally accept to be
# long, since they are less comparable to methods and more comparable to
# modules/classes.
Metrics/BlockLength:
  ExcludedMethods:
    - context
    - describe
    - namespace
    - factory
    - define
    - shared_examples_for
    - shared_examples
    - xcontext
    - xdescribe
```

What do you think?

Reply:
I just disable this check for all RSpec suite because it is too noisy :) Just ignore this one